ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/ovn/local-to-shared-gateway-mode-migration/provision/cucushift-installer-rehearse-vsphere-ipi-ovn-local-to-shared-gateway-mode-migration-provision-chain.metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"path": "cucushift/installer/rehearse/vsphere/ipi/ovn/local-to-shared-gateway-mode-migration/deprovision/cucushift-installer-rehearse-vsphere-ipi-ovn-local-to-shared-gateway-mode-migration-deprovision-chain.yaml",
```

Comment:
And this should be `provision-chain` instead of `deprovision-chain`,

Reply:
same, thanks